scipy/sparse/_coo.py

```diff
@@ -155,8 +156,10 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
                     self._shape = check_shape((M, N))
 
                 idx_dtype = get_index_dtype(maxval=max(self.shape))
-                self.row = np.array(row, copy=copy, dtype=idx_dtype)
-                self.col = np.array(col, copy=copy, dtype=idx_dtype)
+                xp, _ = get_namespace(row)
+                # TODO: Removed dtype and copy, due to being unsupported
```

Comment:
What's meant by "unsupported" here? Documentation for `array_api.asarray` shows that this function should accept dtype and copy kwargs: https://data-apis.org/array-api/latest/API_specification/generated/signatures.creation_functions.asarray.html.